docs/features/shopping.md

```diff
@@ -41,4 +41,4 @@ There are a few more features worth pointing out
 
 1. You can export recipes for use in other applications (Google Keep, etc.) by using the export button
 2. In the export popup you can define a prefix to be put before each row in case an external app requires that
-3. Marking a shopping list as finished will hide it from the shopping list page
\ No newline at end of file
+3. Marking a shopping list as finished will hide it from the shopping list page
```

Comment:
I'm not sure why this line is different, because I made the commit edit in browser and only to line 2. Have not touched line 44